modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/common/components/Button/index.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayButton from '@clayui/button';
+import ClayIcon, {ClayIconSpriteContext} from '@clayui/icon';
+import {forwardRef} from 'react';
+
+import getIconSpriteMap from '../../../utils/getIconSpriteMap';
+
+const ButtonBase = ({
+	buttonText,
+	icon,
+	...props
+}: {
+	buttonText: string;
+	icon: string;
+}) => {
+	return (
+		<>
+			{icon ? (
+				<ClayIconSpriteContext.Provider value={getIconSpriteMap()}>
+					<ClayButton {...props}>
+						<ClayIcon className="mr-1" symbol={icon} />
```

Comment:
É interessante verificar o `icon` aqui, visto que é a única diferença
```suggestion
						icon && (<ClayIcon className="mr-1" symbol={icon} />)
```